discovery/zookeeper/zookeeper.go

```diff
@@ -0,0 +1,220 @@
+package zookeeper
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Minor golang style thing: usually you put the third-party packages last with a line between them and the stdlib packages, so that gofmt sorts them separately.